### PR TITLE
905 feature request element identification name location performance boost

### PR DESCRIPTION
**Pull request by MohabMohie**

### CODE REVIEW

**Comment by MohabMohie** [comment]
wow... 66 files... okay :D 
So... first of all, I'm sorry @MahmoudElSharkawy 
However! let's focus on the good stuff: https://github.com/ShaftHQ/SHAFT_ENGINE/pull/935/commits/326e6a06869cb608baf12e35f6a06588e2849b9f

I separated the REAL changes in their own commit... just 3 files :D 

The other 63 files are just internal classes moving around trying to find their package in the world... please do not disturb them or else they may get violent...

sorry for the comment... it's 02:00AM and I literally woke up from a dream to write this block of code :D

**Comment by MohabMohie** [comment]
@MahmoudElSharkawy clean PR ready for release.
It's been clean since yesterday but I wanted to fix all the broken tests (due to them being old and unmaintained).
This version also includes the fix to the Chrome 111 netty issue which is still compatible with Appium.
After you approve, it will be automatically merged to main, and since I updated the version number, it will also be automatically released.
Kindly approve after the E2E Tests finish execution, and let's push this release... It's a good chance to get all the people who will be hit by this tsunami...

**Review by MahmoudElSharkawy — APPROVED** [review]

**Pull request merged**
